Prompt for 'nuke' again on invalid responses

aqche opened the pull request:
Here is my take on addressing issue #26. Instead of exiting if the user types in an invalid input to the nuke prompt, the user will be notified that the input was invalid and prompted again. A note that you can exit with ^C is added as well.

autero1 commented:
Thanks for the PR @aqche !

The change makes sense, will run the test suite now.

autero1 commented on commands/cli.go:
```diff
@@ -262,19 +262,26 @@ func nukeDefaultSecurityGroups(c *cli.Context, regions []string) error {
 }
 
-func confirmationPrompt(prompt string) (bool, error) {
+func confirmationPrompt(prompt string, maxPrompts int) (bool, error) {
```
Maybe add an explanation here why we're allowing multiple tries.

aqche commented:
@autero1 I added a comment explaining the reasoning behind multiple tries. Also, I removed the unneeded `else` block just to make things slightly cleaner. Thanks!

autero1 commented:
Thanks! I'll merge this in now.
